www-client/iceweasel/iceweasel_bin-128.5.0.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="Unofficial Haiku port of Mozilla Firefox"
+DESCRIPTION="Iceweasel is an open source web browser. \
+It achieves balance between ease of use and customization, catering to the \
+needs of both casual and power users."
+HOMEPAGE="https://github.com/kenz-gelsoft/gecko-dev"
+COPYRIGHT="1995-2024 Mozilla Developers and Contributors"
+LICENSE="MPL v2.0"
+REVISION="1"
+buildTag="HAIKU128_5_20241206"
+SOURCE_URI="https://github.com/kenz-gelsoft/gecko-dev/releases/download/$buildTag/firefox-$portVersion.en-US.haiku-x86_64.tar.bz2#noarchive"
+CHECKSUM_SHA256="311720a084a87da790ed3f992499f939d76e0259387c706336c3c56581bf65f0"
+ADDITIONAL_FILES="iceweasel_icon.hvif"
+
+ARCHITECTURES="x86_64"
+DISABLE_SOURCE_PACKAGE="true"
+
+PROVIDES="
+	iceweasel_bin = $portVersion
+	app:Iceweasel = $portVersion
+	cmd:iceweasel = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libatk_1.0
+	lib:libcairo
+	lib:libcairo_gobject
+	lib:libgdk_3
+	lib:libgdk_pixbuf_2.0
+	lib:libgio-2.0
+	lib:libglib_2.0
+	lib:libgobject-2.0
+	lib:libgtk_3
+	lib:libharfbuzz
+	lib:libintl
+	lib:libnspr4
+	lib:libnss3
+	lib:libpango_1.0
+	lib:libpangocairo_1.0
+	lib:libpng16
+	lib:libz
```

Comment:
For a `_bin` package the exact versions should be specified, so that any "rebuilds" will get the versions of the libs the package was actually built against. If you built the the bin from a recipe, then you can just copy all the lines from the `.PackageInfo` that HaikuPorter generated.

Reply:
Thank you for feedback I'll do that.

Reply:
Adding secondaryArchSuffix for _x86 might be needed though.

Reply:
@korli Thank you for feedback. I have not been built or tested 32bit yet, so I intentionally not including _x86 arch.